FundTrack.WebUI/Controllers/DonateController.cs

```diff
@@ -70,24 +70,18 @@ public string CheckPayment([FromBody] CheckStatusRequestViewModel request)
         /// </summary>
         /// <param name="organizationId">Id of organization</param>
         /// <returns>Organization id and accounts</returns>
-        [HttpGet("GetAccountsForDonate/{organizationId}")]
+        [HttpGet("AccountsForDonate/{organizationId}")]
         public OrganizationDonateAccountsViewModel GetAccountsForDonate(int organizationId)
         {
             return _donateMoneyService.GetAccountForDonation(organizationId);
         }
 
-        [HttpGet("GetOrderId")]
+        [HttpGet("OrderId")]

```

Comment:
"order/id"

Reply:
this method does not accept any argument, but serves to obtain a unique identifier for a financial transaction so /{id} is unnecessary